instrumentation/logback/logback-appender-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/appender/v1_0/OpenTelemetryAppender.java

```diff
@@ -133,12 +159,25 @@ public void setCaptureMdcAttributes(String attributes) {
     }
   }
 
+  /**
+   * Log telemetry is emitted after the initialization of the OpenTelemetry Logback appender with an
+   * {@link OpenTelemetry} object. This setting allows you to modify the size of the cache used to
+   * replay the first logs.
+   */
+  public void setNumLogsCapturedBeforeOtelInstall(int size) {
+    eventsToReplay = new ArrayBlockingQueue<>(size);
```

Comment:
Nit: WDYT about just storing the size here, and only creating the queue in `start()`?